kornia/__init__.py

```diff
@@ -9,8 +9,7 @@
 
 # NOTE: we are going to expose to top level very few things
 from kornia.constants import pi
-from kornia.testing import xla_is_available
```

Comment:
should we plan to do some work on the kornia imports for 0.8.0 release? to try to reduce the number of circular import

Reply:
i think yes -- which possibly means at at least internally not use top level module imports